src/CONST.ts

```diff
@@ -5240,6 +5240,7 @@ const CONST = {
         DATA_TYPES: {
             TRANSACTION: 'transaction',
             REPORT: 'report',
```

Comment:
`transaction` and `report` will be removed in a clean up PR, once the backend starts sending `expense` type